Kajabi/sage-archived
# Sage icon font updates - danger, draft, flag, stack

## Description
This PR updates the Sage icon font and unicode mappings.

## Test notes
The following icons were added:
##### danger 
<img width="127" alt="sage-icon-danger" src="https://user-images.githubusercontent.com/7331038/96631209-07f0d800-12e4-11eb-92d9-c7d621d52867.png">

##### draft
<img width="110" alt="sage-icon-draft" src="https://user-images.githubusercontent.com/7331038/96631247-0e7f4f80-12e4-11eb-833a-89dfc2f78214.png">

##### flag
<img width="106" alt="sage-icon-flag" src="https://user-images.githubusercontent.com/7331038/96631257-150dc700-12e4-11eb-89e6-461d025c8eab.png">

##### stack
<img width="115" alt="sage-icon-stack" src="https://user-images.githubusercontent.com/7331038/96631276-1b03a800-12e4-11eb-9935-efe59a0d6192.png">

### Steps for testing
1. Visit http://localhost:4000/pages/foundations/icon
1. Ensure the added icons appear on the page
1. Ensure all other icons appear as they should
